samples/bluetooth/mesh/light_ctrl/prj.conf

```
@@ -50,6 +49,7 @@ CONFIG_BT_MESH_GATT_PROXY=y
 CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE=y
 CONFIG_BT_MESH_DK_PROV=y
 CONFIG_BT_MESH_NLC_PERF_CONF=y
+CONFIG_BT_MESH_MODEL_EXTENSIONS=y
 CONFIG_BT_MESH_NLC_PERF_LIGHTNESS_CTRL=y
```

Extreme nitpick, but:

```suggestion
CONFIG_BT_MESH_NLC_PERF_LIGHTNESS_CTRL=y
CONFIG_BT_MESH_MODEL_EXTENSIONS=y
```